include/mbedtls/bignum.h

```diff
@@ -135,9 +135,8 @@
             typedef unsigned int mbedtls_t_udbl __attribute__((mode(TI)));
             #define MBEDTLS_HAVE_UDBL
         #endif /* !MBEDTLS_NO_UDBL_DIVISION */
-    #elif defined(__ARMCC_VERSION) && defined(__aarch64__)
+    #elif defined(__aarch64__)
```

Comment:
Ah maybe you would actually prefer to add `__aarch64__` to the `__GNUC__` list because of the `__uint128_t` versus `__attribute__((mode(TI)))`? I could change that. Although I believe that both GCC and Clang have defined `__uint128_t` on 64-bit builds for ages, so I'm not sure if it would matter.